commons-db/src/main/java/com/palantir/nexus/db/sql/SQLString.java

```diff
@@ -367,4 +364,28 @@ public String getKey() {
     public static RegisteredSQLString getRegisteredQueryByKey(FinalSQLString key) {
          return new RegisteredSQLString(key.delegate);
     }
+
+    protected static ImmutableMap<String, FinalSQLString> getCachedUnregistered() {
+        synchronized (cacheLock) {
+            return cachedUnregistered;
+        }
+    }
+
+    protected static void setCachedUnregistered(ImmutableMap<String, FinalSQLString> cachedUnregistered) {
+        synchronized (cacheLock) {
+            SQLString.cachedUnregistered = cachedUnregistered;
+        }
+    }
+
+    protected static ImmutableMap<String, FinalSQLString> getCachedKeyed() {
+        synchronized (cacheLock) {
+            return cachedKeyed;
+        }
+    }
+
+    protected static void setCachedKeyed(ImmutableMap<String, FinalSQLString> cachedKeyed) {
+        synchronized (cacheLock) {
+            SQLString.cachedKeyed = cachedKeyed;
+        }
+    }
```

Comment:
The getter/setter are not being used now. They have been added so that the caches are accessible to subclasses in the future.